integration-test/src/main/java/com/scalar/db/schemaloader/SchemaLoaderIntegrationTestBase.java

```diff
@@ -216,7 +216,7 @@ protected Map<String, Object> getAlteredSchemaJsonMap() {
                         isTimestampTypeSupported()
                             ? ImmutableMap.of("col10", "TIMESTAMPTZ", "col11", "TIMESTAMP")
                             : ImmutableMap.of("col10", "TIMESTAMPTZ"))
-                    .put("col12", "TEXT")
+                    .put("col12", "INT")
```

Comment:
Why do we need to change this type?

Reply:
The `createTableThenAlterTables_ShouldExecuteProperly` will create an index on this column. However, since index creation for a TEXT column fails for Db2 on the CI, I changed the type to INT to avoid disabling this specific test for Db2.